### PR TITLE
WEBUI-429: add vocabulary entry popup title

### PR DESCRIPTION
**Pull request by swarnadipa-dev**

### CODE REVIEW

File: elements/directory/nuxeo-vocabulary-management.js [hunk 1]
```diff
@@ -154,7 +154,7 @@ Polymer({
     </nuxeo-page>
 
     <nuxeo-dialog id="vocabularyEditDialog" with-backdrop>
```
Comment (Gabez0r) [inline elements/directory/nuxeo-vocabulary-management.js:156]: Please note that this dialog is used for both editing and adding new entries, which means that we should display `vocabularyManagement.popup.editEntry` when editing and `vocabularyManagement.popup.addEntry` when creating a new one.

File: elements/directory/nuxeo-vocabulary-management.js [hunk 2]
```diff
@@ -154,7 +154,12 @@ Polymer({
...
-      <h2>[[i18n('vocabularyManagement.popup.editEntry')]]</h2>
+      <template is="dom-if" if="[[_new]]">
```
Comment (richardsd) [inline elements/directory/nuxeo-vocabulary-management.js]: Same applies to you @swarnadipa-dev. See https://github.com/nuxeo/nuxeo-web-ui/pull/1516#discussion_r920079671 for details.

**Review by richardsd — DISMISSED** [review]

File: elements/directory/nuxeo-vocabulary-management.js [hunk 3]
```diff
@@ -154,7 +154,7 @@ Polymer({
     </nuxeo-page>
 
     <nuxeo-dialog id="vocabularyEditDialog" with-backdrop>
-      <h2>[[i18n('vocabularyManagement.popup.editEntry')]]</h2>
+      <h2>[[_entryLabel(_new)]]</h2>
```
Comment (richardsd) [inline elements/directory/nuxeo-vocabulary-management.js]: You could probably rename the method to something easier to understand the meaning. See https://github.com/nuxeo/nuxeo-web-ui/pull/1516/files#diff-91c3d1d5918e7a6444f10b8efaf01283d38cb9bfc912e6a2e9e0b683e596b92fR102 as an example too.

**Review by richardsd — COMMENTED** [review]
@swarnadipa-dev don't forget to review my comment and reword the commit history, because it is not following the guidelines.
Check https://doc.nuxeo.com/nxdoc/contributing-to-nuxeo/#contribution-principles for more details about this 🙂 Nice work nevertheless

**Review by Gabez0r — COMMENTED** [review]
Please note that there are 4 commits, 3 of which (the first three) do not add anything to the history, (and also, they don't follow our commit guidelines). Please squash them.
image

**Review by SnehalNaik-DEV — DISMISSED** [review]

**Comment by Gabez0r** [comment]
Also please amend your author information, which is pointing to `Global <global@DEL1-LMC-N77370.local>`. This is actually required to sign the CLA I think.

**Review by rakeshkumar1019 — COMMENTED** [review]
looks good

**Review by Gabez0r — DISMISSED** [review]
Good job. Please do not forget to sign the CLA agreement. 
image

**Review by SnehalNaik-DEV — DISMISSED** [review]
Seems good!

**Review by Gabez0r — APPROVED** [review]

**Review by SnehalNaik-DEV — APPROVED** [review]
Looks good!

**Pull request merged**